packages/shylu/shylu_node/basker/src/shylubasker_def.hpp

```diff
@@ -389,47 +424,79 @@ namespace BaskerNS
 
       #ifdef BASKER_TIMER
       init_time += timer_init.seconds();
-      std::cout << "Basker Symbolic matrix init time: " << init_time << std::endl;
+      std::cout << std::endl << "Basker Symbolic matrix init time: " << init_time << std::endl;
+      /*{
```

Comment:
For a later cleanup follow up, we may want to add a macro that can be enabled to guard for this type output